packages/legend-extension-application-studio-query-builder/src/components/QueryBuilder_LegendStudioApplicationPlugin.tsx

```diff
@@ -220,12 +220,12 @@ export class QueryBuilder_LegendStudioApplicationPlugin
                   }),
                 );
                 if (queryBuilderExtension.mode) {
-                  queryBuilderExtension.queryBuilderState.changeClass(element);
                   queryBuilderExtension.queryBuilderState.initialize(
                     editorStore.graphManagerState.graphManager.createGetAllRawLambda(
                       element,
                     ),
                   );
+                  queryBuilderExtension.queryBuilderState.changeClass(element);
```

Comment:
You would need to look at `DataSpaceQueryEditorStore`

Reply:
I think we are good and don't need any changes in `DataSpaceQueryEditorStore`

Reply:
As discussed, I don't think we need it